apis/fluentbit/v1alpha2/plugins/output/s3_types.go

```diff
@@ -0,0 +1,157 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/v2/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/v2/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// The S3 output plugin, allows to flush your records into a S3 time series database. <br />
+// **For full documentation, refer to https://docs.fluentbit.io/manual/pipeline/outputs/s3**
+type S3 struct {
+	// The AWS region of your S3 bucket
+	Region string `json:"region"`
+	// S3 Bucket name
+	Bucket string `json:"bucket"`
+	// Specify the name of the time key in the output record. To disable the time key just set the value to false.
+	JsonDateKey string `json:"json_date_key,omitempty"`
+	// Specify the format of the date. Supported formats are double, epoch, iso8601 (eg: 2018-05-30T09:39:52.000681Z) and java_sql_timestamp (eg: 2018-05-30 09:39:52.000681)
+	JsonDateFormat string `json:"json_date_format,omitempty"`
+	// Specifies the size of files in S3. Minimum size is 1M. With use_put_object On the maximum size is 1G. With multipart upload mode, the maximum size is 50G.
+	TotalFileSize string `json:"total_file_size,omitempty"`
```

Comment:
This naming scheme is not in line with the original naming scheme of the `fluent-operator`.
```
TotalFileSize  string `json:"TotalFileSize,omitempty"`
```
https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/output/elasticsearch_types.go#L43
What do you think?

Reply:
You're right, I fixed it
Thanks